### DISCUSSION TITLE
Nuxt and CDN's

### DISCUSSION THREAD

**Discussion by monkemedia** [nuxt/nuxt#20957] [Questions]
I am constantly trying to improve the performance of my website. Using seositecheckup.com, they strong recommend that i use a CDN to dish out the `_nuxt` files (ie. `/_nuxt/i18n.config.9ea82aac.js`). First of all, is this possible? If so how do I go about it?

Site is hosted on vercel, while also using Cloudflare.

**Comment by warflash** [comment]
If you're using vercel + cf then vercel should already serve static assets from a cache + you can configure cloudflare to cache /_nuxt/ assets by setting a Cache Rule for it

**Reply by monkemedia** [reply]
Thank you Warflash. Yes I am currently serving static files such as images via Vercel's CDN. I have had a play with cloudflare and added the new Cache Rules, sadly it doesnt look like CDN routes are added to the `_nuxt` paths. Is this to be expected?
